apps/gitness/src/pages-v2/webhook/stores/webhook-store.tsx

```diff
@@ -0,0 +1,38 @@
+import { create } from 'zustand'
+
+import { ListRepoWebhooksOkResponse } from '@harnessio/code-service-client'
+import { WebhookType } from '@harnessio/ui/views'
+
+import { timeAgoFromEpochTime } from '../../../pages/pipeline-edit/utils/time-utils'
+import { PageResponseHeader } from '../../../types'
+
+interface WebhookStore {
```

Comment:
the interface should be in `ui/views` so that we can mock this data correctly in the new playground

Reply:
done